05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,4 +1,58 @@
-.lawn {
+#checkpoint{
+
+}
+
+.image {
+  width: 500px;
+  padding-right: 5px;
+  float: left;
+}
+
+.copy{
+  width: 500px;
+  height: 400px;
+  background-color: RGB(240,240,240);
+  align-items: center;
+  padding: 7% 0;
+  float: left;
```

Comment:
if the image and copy divs have the same width and float, can you factor these out?